thirdeye/thirdeye-frontend/app/actions/events.js

```diff
@@ -87,26 +91,28 @@ const assignEventTimeInfo = (event, anomalyStart, anomalyEnd, baselineStart, bas
 
   const relStart = event.start - anomalyStart;
 
-  var relEnd = Infinity;
-  var relDuration = Infinity;
+  let relEnd = Infinity;
+  let relDuration = Infinity;
   if (event.end > 0 && moment().isAfter(moment(event.end).add(1, 'minute'))) {
     relEnd = event.end - anomalyStart;
     relDuration = event.end - event.start;
   }
 
-  var isBaseline = false;
-  var displayStart = event.start;
-  var displayEnd = event.end;
-  if (baselineStart <= displayStart && displayStart < baselineEnd) {
+  let isBaseline = false;
+  let displayStart = event.start;
```

Comment:
fyi, you can use es6's destructuring to do this: 
```
let { 
  start : displayStart, 
  end: displayEnd,
  label: displayLabel
} = event; 
``` 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring